Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN curl -sS -o /usr/share/keyrings/torproject.asc https://deb.torproject.org/to
 USER debian-tor
 
 COPY tor-entrypoint /usr/local/bin/tor-entrypoint
-COPY tor-generate-onion /usr/local/bin/tor-generate-onion
+COPY generate-service-keypair /usr/local/bin/generate-service-keypair
```

Comment:
The filename was renamed to `gen-service-keypair`, not `generate-service-keypair`

Reply:
Fixed, thanks!